internal/server/main.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+
+	"go_Playlist_gRPC/internal/playlist"
+	pb "go_Playlist_gRPC/internal/proto/music_player"
+)
+
+type server struct {
+	pb.UnimplementedMusicPlayerServer
+	Cmd chan playlist.Commands
+}
+
+func (s server) Play(ctx context.Context, e *pb.Empty) (*pb.PlaylistResponse, error) {
```

Comment:
I suggest moving the `server` structure to the heap and having reference methods